### PR TITLE
Update the README for Quartz Software to make more sense

### LINKED ISSUE #1: Update the README for Quartz Software to make more sense

### PR DESCRIPTION
**Pull request by clarkematthew**
Fixes #1

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Hi there 👋
-
+We are Quartz
```
Comment (jacobaclarke) [inline profile/README.md]: This is kind of boring. Can you make it better?

**Review by jacobaclarke — COMMENTED** [review]
See my comments

File: profile/README.md (hunk 1 above)
Comment (clarkematthew) [reply profile/README.md]: Hey I fixed it boss.

**Review by jacobaclarke — COMMENTED** [review]
This looks great!

**Pull request merged**